docs/reference/aggregations/bucket/datehistogram-aggregation.asciidoc

```diff
@@ -425,7 +425,7 @@ POST /sales/_search?size=0
 --------------------------------------------------
 // CONSOLE
 // TEST[setup:sales]
-// TEST[warning:The joda time api for doc values is deprecated. Use -Des.scripting.use_java_time=true to use the java time api for date field doc values]
+// TEST[warning:getDayOfWeek() is incompatible with the java time api. Use getDayOfWeekEnum().getValue()."]
```

Comment:
Maybe "the return type of getDayOfWeek will change in 7.0. Call getDayOfWeekEnum().getValue() to be compatible with 7.0."